e2e/test/scenarios/visualizations-tabular/object_detail.cy.spec.js

```diff
@@ -101,30 +101,18 @@ describe("scenarios > question > object details", { tags: "@slow" }, () => {
             alias: "Products",
           },
         ],
-        limit: 5,
+        limit: 2,
       },
     };
 
     createQuestion(questionDetails, { visitQuestion: true });
+    cy.findByTestId("question-row-count").should("have.text", "Showing 2 rows");
 
-    cy.log("check click on 1st row");
-
-    cy.get("[data-testid=cell-data]").contains("37.65").realHover();
-    cy.findByTestId("detail-shortcut").findByRole("button").click();
-
-    cy.findByTestId("object-detail").within(() => {
-      cy.get("h2").should("contain", "Order").should("contain", 1);
-      cy.findByTestId("object-detail-close-button").click();
-    });
-
-    cy.log("check click on 3rd row");
-
-    cy.get("[data-testid=cell-data]").contains("52.72").realHover();
-    cy.findByTestId("detail-shortcut").findByRole("button").click();
-
+    cy.log("Check object details for the first row");
```

Comment:
We need to open the object detail modal at least twice, see repro steps for #39477

Reply:
In that case we might have a regression, and not a flake.
It's really hard to determine whether this test fails or flakes.

I'll give it another try.

Reply:
Updated the test and re-ran the stress-test.
Updated the PR description as well.
https://github.com/metabase/metabase/pull/45973/commits/369214007642a275d133f75c2149ef4d83e89d1a